src/data/queries/contactInfo.ts

```diff
@@ -0,0 +1,57 @@
+import { QueryParams, QueryFormatter } from 'next-drupal-query'
+import { ParagraphContactInformation } from '@/types/drupal/paragraph'
+import { ContactInfo } from '@/types/formatted/contactInfo'
+import { queries } from '.'
+
+export const params: QueryParams<null> = () => {
+  return queries.getParams().addInclude([
+    'field_additional_contact', // can be paragraph--phone_number or paragraph--email_contact
+    'field_contact_default',
+    'field_contact_default.field_office',
+    'field_benefit_hub_contacts',
+    'field_benefit_hub_contacts.field_support_services', // this is the additional contact info for a Benefit Hub, node--support_service
+  ])
+}
+
+// paragraph--contact_information is essentially a wrapper paragraph around several other entity references
+export const formatter: QueryFormatter<
+  ParagraphContactInformation,
+  ContactInfo
+> = (entity: ParagraphContactInformation) => {
+  const additionalContactType = entity.field_additional_contact?.type
+  let additionalContact = null
+
+  switch (additionalContactType) {
+    case 'paragraph--email_contact':
```

Comment:
`paragraph--email_contact` should maybe be replaced with an enum/const reference, but this can be cleaned up when we revisit the type cleanup as a whole.

Reply:
sounds good. going to merge as-is for now just to keep things moving / make these available for the layouts that need them knowing it will be iterated on

Reply:
Though, this might go away. See comment below about how we might format this paragraph.